bigtable/tests/unit/test_table.py

```diff
@@ -411,6 +407,60 @@ def _list_column_families_helper(self):
     def test_list_column_families(self):
         self._list_column_families_helper()
 
+    def _test_get_cluster_states_helper(self):
```

Comment:
Please rename this into `def test_get_cluster_states(self)` and remove the reference [below ](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/5790/files#diff-d495c61ac7ca28885a9cb09130c527c1R462)